concordium-std/src/impls.rs

```diff
@@ -2382,6 +2476,19 @@ impl HasHost<ExternStateApi> for ExternLowLevelHost {
         parse_upgrade_response_code(response)
     }
 
+    fn account_public_keys(&self, address: AccountAddress) -> QueryAccountPublicKeysResult {
+        query_account_public_keys_worker(address)
+    }
+
+    fn check_account_signature(
+        &self,
+        address: AccountAddress,
+        signatures: &AccountSignatures,
+        data: &[u8],
+    ) -> CheckAccountSignatureResult {
+        check_account_signature_worker(address, signatures, data)
+    }
+
```

Comment:
duplicated

Reply:
Yes, this is for the two different Host implementations, the first one is for low-level, and another one for the "normal" host.

That is why there is a helper `check_account_signature_worker`.